DependencyInjection/Configuration.php

```diff
@@ -40,10 +40,24 @@ public function getConfigTreeBuilder()
                     ->requiresAtLeastOneElement()
                     ->useAttributeAsKey('name')
                     ->prototype('array')
-                        ->children()
+						->children()
                             ->scalarNode('title')->isRequired()->end()
                             ->scalarNode('description')->isRequired()->end()
-                            ->scalarNode('link')->isRequired()->end()
+							->arrayNode('link')
+								->isRequired()
+								->beforeNormalization()
+									->ifString()
+									->then(function ($v) { return [ 'direct' => $v]; })
+								->end()
+								->children()
+									->scalarNode('route_name')->end()
+									->arrayNode('route_params')
+										->useAttributeAsKey('key')
+										->prototype('scalar')->end()
+									->end()
+									->scalarNode('direct')->end()
```

Comment:
I suggest giving the `direct` configuration name a more understanding name such `uri`. I think it should be better, what do you think?